src/amo/utils/index.js

```diff
@@ -166,15 +166,15 @@ export const stripLangFromAmoUrl = ({
   return urlString;
 };
 
-export function getClientConfig(_config: typeof config) {
+export function getClientConfig(_config: typeof config): { ... } {
```

Comment:
What does this notation mean?

Reply:
I'm not sure myself. I think it's a way of specifying an object, but I don't like it. It's something the codemod added and I'm removing it as I work through the cleanup.